### PR TITLE
[INCR-104] Add trove classifiers to crowdsourcehinter

### PR DESCRIPTION
**Pull request by cclauss**
Fixes https://openedx.atlassian.net/browse/INCR-104

Just like solashirai/crowdsourcehinter#12

### CODE REVIEW

**Comment by edx-webhook** [comment]
Thanks for the pull request, @cclauss! I've created [OSPR-3212](https://openedx.atlassian.net/browse/OSPR-3212) to keep track of it in JIRA. JIRA is a place for product owners to prioritize feature reviews by the engineering development teams.

Feel free to add as much of the following information to the ticket:
- supporting documentation
- edx-code email threads
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will still be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).

**Comment by natabene** [comment]
@cclauss Thank you for contribution. @edx/incr-reviewers Can you give this a look?

File: setup.py [hunk 1]
```diff
@@ -37,4 +37,14 @@ def package_data(pkg, roots):
         ]
     },
     package_data=package_data("crowdsourcehinter", ["static", "public"]),
+    classifiers=[
+        'Framework :: Django :: 1.11',
+        'Programming Language :: Python',
+        'Programming Language :: Python :: 2',
+        'Programming Language :: Python :: 2.7',
+        'Programming Language :: Python :: 3',
+        'Programming Language :: Python :: 3.6',
```
Comment (jmbowman) [inline setup.py]: It looks like we're not actually testing the 3.6 support yet; could you uncomment that from the Travis configuration while you're at it?

Comment (cclauss) [reply setup.py]: That was done in #3 but I added it here as well.

Comment (jmbowman) [reply setup.py]: Thanks, didn't see that one; I'll see about getting that cleaned up.

**Comment by edx-webhook** [comment]
@cclauss 🎉 Your pull request was merged!

Please take a moment to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=edx/crowdsourcehinter&entry.867055334=https://github.com/edx/crowdsourcehinter/pull/4&entry.1484655318=https://github.com/cclauss&entry.752974735=2019-03-15+11:13&entry.1917517419=2019-03-19+18:19&entry.2133058324=Yes) so we can improve your experience in the future.

**Pull request merged**